test/unit_tests/utils/test_generic_utils.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+# Copyright 2018-2023 EvaDB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+from evadb.utils.generic_utils import string_comparison_case_insensitive
+
+
+class GenericUtilsTests(unittest.TestCase):
```

Comment:
Ah I was wondering whether to create a new test file add in a new file or not. Thanks for this, makes it clear